Gemfile

```diff
@@ -3,21 +3,21 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'rake', '~> 11.1', :group => [:development, :test]
+gem 'rake', :group => [:development, :test]
 
 group :development do
```

Comment:
Hm, sorry, but imho that's not a good idea. The only purpose of the `Gemfile` is to provide the exactly the same working and production environment for your team mates. You may want to update the versions, but you have to control them. We need versions!

Reply:
Yes you are right. I was investigating the travis failing and thought it was from an old rake version.
Turns out that wasn't the problem.